src/lib/dispatcher/kafka-dispatcher.js

```diff
@@ -28,14 +28,16 @@ class KafkaDispatcher extends winston.Transport {
         console.error("Unable to connect to kafka", err.message);
       });
   }
-  async log(level, msg, mid, callback) {
+  async log(level, msg, kafkaTopics, callback) {
     try {
+      console.log(`pushing events to topic '${kafkaTopics[0]}'...`);
       await this.producer.send({
-        topic: this.options.topics.create,
+        topic: kafkaTopics[0],
```

Comment:
Use the loadash lib.  `_.head(kafkaTopics)` and You need to ensure the sequence of the topics should be correct else it will push to different topic right? Can we solve with map format?